src/argilla/client/feedback/dataset/remote/dataset.py

```diff
@@ -105,6 +105,14 @@ def _client(self) -> "httpx.Client":
         """Returns the `httpx.Client` instance that will be used to send requests to Argilla."""
         return self.dataset._client
 
+    @property
+    def _question_id_to_name(self) -> Dict["UUID", str]:
+        return self.dataset._question_id_to_name_id
+
+    @property
+    def _question_name_to_id(self) -> Dict[str, "UUID"]:
+        return self.dataset._question_name_to_id
+
```

Comment:
Do we actually need to wrap those properties under the same name? IMO we can just re-use those from `self.dataset`